conf/layer.conf

```diff
@@ -32,23 +29,10 @@ LICENSE_PATH += "${LAYERDIR}/licenses"
 LICENSE_FLAGS_WHITELIST += "commercial_mpg123"
```

Comment:
IMO: Those whitelists shouldn't be there too. We shouldn't whitelist the licences at all here

Reply:
moved to platform layer